README.md

```diff
@@ -153,7 +153,75 @@ Example output:
 
 # Usage in CI Systems
 
-> 🚧 COMING SOON 🚧
+> 🚧 COMING SOON: native docker image & GitHub action 🚧
+
+## GitHub Actions workflows
+
+Here is an example of a simaple GitHub Actions workflow that uses `kubectl-validate` to validate Kubernetes manifests.
+This workflow will run on every pull request to the `main` branch and will fail if any of the manifests in the dir `k8s-manifest/` are invalid.
+
+```yaml
+name: kubectl-validate
+on:
+  pull_request:
+    branches:
+      - main
+  
+env:
+  MANIFESTS_PATH: 'k8s-manifest/*.yaml'
+  SCHEMA_VERSION: '1.23'
```

Comment:
I find this more confusing than helping, can we just keep the  values directly in the command?

Reply:
NP